FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Tan Tan on 5/31/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct FeedImageMapper {
+	struct Root: Decodable {
+		var items: [FeedImageResult] = []
+
+		struct FeedImageResult: Decodable {
+			public let image_id: UUID
+			public let image_desc: String?
+			public let image_loc: String?
+			public let image_url: URL
```

Comment:
Those properties are only used in the `FeedImageMapper` which has an `internal` access control. So making them public won't have any effect. You can remove the public keyword:

```suggestion
			let image_id: UUID
			let image_desc: String?
			let image_loc: String?
			let image_url: URL
```